src/core/include/cesium/omniverse/GltfUtil.h

```diff
@@ -27,6 +27,9 @@ enum class AlphaMode : int {
     BLEND = 2,
 };
 
+enum class OverlayType { ION = 0, POLYGON = 1 };
+enum class OverlayRenderPipe { OVERLAY = 0, CLIPPING = 1 };
```

Comment:
Could you add a comma after `POLYGON = 1` and `CLIPPING = 1`? `clang-format` will format them more nicely

Reply:
Interesting. Yeah, it did re-format it. I mildly prefer the concision of what's above, but let's change it for consistency across the code base.